browser/mapping.go

```diff
@@ -31,11 +31,12 @@ type mapping = map[string]any
 // See issue #661 for more details.
 func mapBrowserToGoja(vu moduleVU) *goja.Object {
 	var (
-		rt          = vu.Runtime()
-		obj         = rt.NewObject()
-		browserType = chromium.NewBrowserType(vu)
+		rt                     = vu.Runtime()
+		obj                    = rt.NewObject()
+		wsURL, isRemoteBrowser = k6ext.IsRemoteBrowser()
+		browserType            = chromium.NewBrowserType(vu)
 	)
-	for k, v := range mapBrowserType(vu, browserType) {
+	for k, v := range mapBrowserType(vu, browserType, wsURL, isRemoteBrowser) {
```

Comment:
The distinction that I had in mind was in the case that there is an exported `K6_BROWSER_WS_URL` variable, but with value `""` (void string). In that case `isRemoteBrowser` will be `true` but `wsURL` will be `""`. What should we do in that case? Personally I thought that it was important to make this distinction because if the env var is set we can assume that the browser was expected to be remote (even if maybe there was an error setting the value), and therefore continue with the `connect` process, which will eventually fail. Otherwise, we only call `launch` if the env var is unset, not if its value is a void string.

What is your take on this?